easybuild/easyconfigs/s/Simpack/Simpack-2017.eb

```diff
@@ -0,0 +1,20 @@
+# Author: Alexi Rivera
+# C3SE / Chalmers University of Technology
+
+name = 'Simpack'
+version = '2017'
+
+homepage = 'www.simpack.com'
+
+description = """ SIMPACK is a general purpose Multi-Body Simulation (MBS) software which
+ is used for the dynamic analysis of any mechanical or mechatronic system. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCE_TAR_GZ]
+
+import os
```

Comment:
We should try and get rid of `import` statements in eaysconfig files, but we don't have a better alternative yet for this...